article/app/views/liveblog/liveBlogBodyContent.scala.html

```diff
@@ -14,8 +14,8 @@
         @if(blockGroup.length == 5 && index < 8) {
             <div class="block amp-ad-container amp-ad-container--live-blog">
                 <amp-ad width="300" height="250" type="doubleclick"
-                json=@AmpAd.adTargetingJson(article, request.path, Edition(request).id.toLowerCase()).toString()
-                data-slot=@AmpAd.buildDataSlot(article)>
+                json=@AmpAd(article, request.path, Edition(request).id.toLowerCase()).toJson().toString()
```

Comment:
I guess you can also add the `toString` method in the AmpAd class, so you don't need to explecitely do `toJson().toString` every time you need a string